answers/Inheritance/91b33c9d-8c7d-4a40-ba1d-fc1b16a641eb/Student.php

```diff
@@ -0,0 +1,27 @@
+<?php
+require_once(__DIR__ . '/Person.php');
+class Student extends Person
+{
+    private $course;
+    	
+    public function __construct($name, $age, $course)
+    {
+        parent::__construct($name, $age);
+
+        $this->course = $course;
+    }
+    
+    public function getCourse()
+    {
+        return $this->course;
+    }
+
+    public function display()
+    {
+        echo parent::getName() . " is taking up " . $this->course . ".";
+    }
+}
+
+$student = new Student("John", 20, "BSCS");
+$student->display();
```

Comment:
Can we move this outside of this class? Create a new file say `index.php` which does
```
require_once(__DIR__ . '/Student.php');

$student = new Student(...);
$student->display();
```